tests/playTests.sh

```diff
@@ -220,6 +225,14 @@ ZSTD_CLEVEL=3a7 zstd -f tmp # malformed env var, warn and revert to default sett
 ZSTD_CLEVEL=50000000000 zstd -f tmp # numeric value too large, warn and revert to default setting
 println "test : override ZSTD_CLEVEL with command line option"
 ZSTD_CLEVEL=12  zstd --fast=3 -f tmp # overridden by command line option
+# restore original value or unset
+if [ -n "${ORIGINAL_ZSTD_CLEVEL}" ]; then
+  ZSTD_CLEVEL=$ORIGINAL_ZSTD_CLEVEL
+else
+  unset ZSTD_CLEVEL
```

Comment:
If the goal is just to ensure that no environment variables bleed between tests, they maybe just this line : 
`unset ZSTD_CLEVEL` 
would be enough ?

I don't see a good reason to preserve previous value of `ZSTD_CLEVEL` and re-generate it for future tests.
Ideally, we would like each test to be as autonomous as possible, to better control test conditions.
This `playtests.sh` script doesn't always respect this rule, sometimes to better re-employ resources, but I believe it would be preferable to lean towards "more autonomous tests".

Reply:
We can definitely do that, but if we do we should probably start the script with an unset so the environment variables will be disabled for the previous tests as well. Will get to it later today.